var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -54,9 +56,20 @@ class Libfabric(AutotoolsPackage):
        multi=True
     )
 
+    variant('spackfabrics', 
+            default=False,
+            description=("For any fabric in list of "
+                         "enabled fabrics, use spack "
+                         "dependencies instead of "
+                         "system libraries")
+    )
+
+    depends_on('rdma-core', when='+spackfabrics fabrics=verbs')
```

Comment:
please update ucx package as well 
 var/spack/repos/builtin/packages/ucx/package.py

it depends on rdma-core

Reply:
I added the rdma-core dependency to ucx. It was building successfully without it, but I think disabling verbs support. With the dependency added, linked libraries look good.